scripts/base/build_qt.sh

```diff
@@ -26,6 +27,23 @@ else
     tar xf "$DOWNLOADS_DIR/qt-${ASWF_QT_VERSION}.tar.xz"
     mv "qt-everywhere-src-${ASWF_QT_VERSION}" qt-src
     cd qt-src
+    if [[ $QT_MAJOR_MINOR == 5.15 ]]; then
```

Comment:
Should this test be >= 5.15, or do we expect these options to be specific to that Qt version?

Reply:
You're right, but as there's a good chance that next Qt upgrade would move to Qt6 and would use CMake instead, at that point I'm feeling too lazy to add number conversions to do the proper comparison...